src/index.ts

```diff
@@ -245,6 +249,22 @@ export const htmlPlugin = (configuration: Configuration = { files: [], }): esbui
                         document.head.appendChild(linkTag)
                     }
 
+                    for (const script of htmlFileConfiguration?.extraScripts || []) {
+                        const scriptTag = document.createElement('script')
+                        if (typeof script === 'string') {
+                            scriptTag.setAttribute('src', script)
+                        } else {
+                            scriptTag.setAttribute('src', script.src)
```

Comment:
We could also add the attributes set by scriptLoading here, but not sure if this is a good idea.

Reply:
Not sure that's necessary, I added the tags (now attrs) prop so people can customize it for whatever their needs may be. I imagine people using this will probably have specific scripts in mind and don't necessarily want to be locked in to all of them being defer/module/etc?

Reply:
Sounds reasonable :)